.gitignore

```diff
@@ -7,3 +7,5 @@ node_modules
 .env
 /styleguide
 .coverage
+src/views/.DS_Store
+config.env
```

Comment:
What is this for?

Reply:
Ah that was an extra file I created. I'll remove everything that isn't the main modified file then.